src/gatsby-plugin-theme-ui/index.js

```diff
@@ -38,7 +41,21 @@ export default {
     root: {
       fontFamily: 'body',
       fontWeight: 'body',
-      height: '100%',
+      minHeight: '100vh',
```

Comment:
This breaks scrollbar styles, since the `root` element is no longer the scroll root.